infrastructure/parameters/dev.parameters.json

```diff
@@ -28,6 +28,12 @@
     },
     "deploySubnets": {
       "value": true
+    },
+    "blobDeleteRetentionEnabled": {
+      "value": true
```

Comment:
Shouldnt need this set as its default is true

Reply:
I'm assuming that we'll set this to false in dev once we've tested so more of a placeholder.